pkg/service/manipulator.go

```diff
@@ -83,18 +74,27 @@ func (m *manipulator) Process(spec ProcessSpec) ([]byte, error) {
 		data = m.processor.GrayScale(data)
 		trackDuration(grayScaleDurationKey, t, spec)
 	}
-
 	if radius := CleanFloat(params[blur], 1000); radius > 0 {
 		t = time.Now()
 		data = m.processor.Blur(data, radius)
 		trackDuration(blurDurationKey, t, spec)
 	}
 
-	if params[auto] == compress {
-		orientation, _ := native.GetOrientation(bytes.NewReader(spec.ImageData))
-		t = time.Now()
-		data = m.processor.FixOrientation(data, orientation)
-		trackDuration(fixOrientationKey, t, spec)
+	autos := strings.Split(params[auto], ",")
+	for _, a := range autos {
+		if a == compress {
+			orientation, _ := native.GetOrientation(bytes.NewReader(spec.ImageData))
+			t = time.Now()
+			data = m.processor.FixOrientation(data, orientation)
+			trackDuration(fixOrientationKey, t, spec)
+		} else if a == format {
```

Comment:
Is there a possibility of `auto=compress,format`? If not, then please ignore this comment.

Reply:
yes, it is. in that case, we will have to fix the orientation and return the image as webp whenever possible. is there any concern?

Reply:
I don't see any concern but current flow has an `else` so only one value would work.

Reply:
The reason I use `else if` is because we should process the image as webP only if `format` is being sent. With `else`, any other parameter (except `compress`) will be marked as webP param.